### DISCUSSION TITLE
Compatibility With MiniDsp 2x4

### DISCUSSION THREAD

**Discussion by andresdalmazzo** [mrene/minidsp-rs#186] [Q&A] [answered]
Hi, im starting a project and i have a MiniDSP 2x4 Balanced, (non HD). I know there is a compatibility list, but im not sure if the MiniDsp 2x4 (non HD) is supported.
In advanced, thanks for your answers.

**Comment by mrene** [comment, marked as answer]
As far as I know the 2x4 uses a different protocol, it's not currently supported.
